cloudbuild.yaml

```diff
@@ -152,11 +133,12 @@ steps:
 # - https://pantheon.corp.google.com/storage/browser/artifacts.oak-ci.appspot.com/test/?project=oak-ci
 # - https://cloud.google.com/cloud-build/docs/configuring-builds/store-images-artifacts#storing_artifacts_in
 # TODO: Finalize artifact location.
-artifacts:
-  objects:
-    location: gs://artifacts.oak-ci.appspot.com/test
-    paths:
-      - ./liboak_enclave_unsigned.so
+# TODO(#709): get example Docker image working with dev server
+#artifacts:
```

Comment:
Perhaps we should keep uploading the compiled oak_runtime (dev) binary? Not sure it's particularly useful by itself though.

Reply:
I don't know anything about how the artifact upload stuff works, so it seemed easiest just to disable it for the moment; maybe @ipetr0v can explain and/or fix it for me?